homeassistant/components/notify/message_bird.py

```diff
@@ -0,0 +1,77 @@
+"""
+MessageBird platform for notify component.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.message_bird/
+"""
+import logging
+
+from homeassistant.components.notify import (
+    ATTR_TARGET, BaseNotificationService)
+from homeassistant.const import CONF_API_KEY
+
+CONF_SENDER = 'sender'
+
+_LOGGER = logging.getLogger(__name__)
+REQUIREMENTS = ['messagebird==1.1.1']
+
+
+def is_valid_sender(sender):
+    """Test if the sender config option is valid."""
+    length = len(sender)
+    if length > 1:
+        if sender[0] == '+':
+            return sender[1:].isdigit()
+        elif length <= 11:
+            return sender.isalpha()
+    return False
+
+
+# pylint: disable=unused-argument
+def get_service(hass, config):
+    """Get the MessageBird notification service."""
+    from messagebird import Client
+
+    if CONF_API_KEY not in config:
+        _LOGGER.error("Unable to find config key '%s'", CONF_API_KEY)
+        return None
+
+    sender = config.get(CONF_SENDER, 'HA')
```

Comment:
Your [PR](https://github.com/balloob/home-assistant.io/pull/308) is not mentioning `sender` as optional input.